main

```diff
@@ -0,0 +1,124 @@
+import asdasd
+import telebot
+import requests
+from peewee import *
+from asdasd import *
+import datfile
+parameters = {
+    #'q': 'dollar', # query phrase  # maximum is 100
+    'country': 'gb',
+    #'source': None,
+    'category': 'business',
+    'apiKey': '0b19c9bde21541859e8176ed2060de79' # your own API key
+}
+import json
+from telebot import types
+#import TicTacToe
+url = 'https://newsapi.org/v2/top-headlines?'
+parameters = {
+    #'q': 'dollar', # query phrase  # maximum is 100
+    'country': 'gb',
+    #'source': None,
+    'category': 'business',
+    'apiKey': '0b19c9bde21541859e8176ed2060de79' # your own API key
+}
+db = SqliteDatabase('db/database.db')
+class sus():
+    id = PrimaryKeyField(unique=True)
+    country = CharField()
+    category = CharField()
+    ever = IntegerField()
+
+bot = telebot.TeleBot('5247888467:AAH2hPIdtmH26P3mhdCKkpSua9Pt44mYoY0');
+
+
+@bot.message_handler(content_types=['text'])
+def get_text_messages(message):
+    if message.text == "/start":
+        bot.send_message(message.from_user.id, "Hello, I am news bot. Please enter your username:")
+        bot.register_next_step_handler(message, start_search0);
+
+def start_search0(message):
+    global nick
+    nick = message.text
+    if asdasd.check_name(nick):
+        asdasd.write_player(nick)
+        bot.send_message(message.from_user.id, "I'm glad to see you, "+ nick +"\n/get_some_news — choose the country whose news you want to receive,\n/help — Repeats this message,\n/stats - Get stats")
```

Comment:
It's better to make lines shorter and to use python multiline string (here and below)